src/forwarders.interfaces.ts

```diff
@@ -0,0 +1,45 @@
+import { SDKEvent } from './sdkRuntimeModels';
+import { Dictionary } from './utils';
+import { IKitConfigs } from './configAPIClient';
+
+// The state of the kit when accessed via window.KitName via CDN
+// or imported as an NPM package
+export interface UnregisteredKit {
+    register(config): void;
+}
+
+// The state of the kit after being added to forwarderConstructors in the CDN 
+// or after registered to SDKConfig.kits via NPM
+export interface RegisteredKit {
+    constructor(): void;
+    name: string;
+    getId(): number;
+}
+
+// The state of the kit aftering being configured. This is what the kit looks like when acted on.
+export interface ConfiguredKit
+    extends Omit<IKitConfigs, 'isDebugString' | 'hasDebugString'> {
+    common: Dictionary;
+    init(settings, service, testMode, trackerId, userAttributes, userIdentities, appVersion, appName, customFlags, clientId): string;
+    onIdentifyComplete(user, filteredIdentityRequest): string | KitMappedMethodFailure;
```

Comment:
for these signatures, `user` should be `user: MPUser`. But what is `filteredIdentityRequest`?

Also, based on usage, shouldn't the return type be `boolean` not `string`?

Lastly, can you provide a link to a sample of how this is used, to make it easier for me to grok how this is implemented?